checkpoint_engine/ps.py

```diff
@@ -1224,6 +1225,7 @@ def _update_per_bucket(
         socket.send_pyobj(handle)
 
         gidx = 0
+        ret_code = torch.tensor(0, device=self.device_manager.device_type)
```

Comment:
Maybe use `torch.zeros(tuple(), device=self.device_manager.device_type, dtype=torch.int64)` to avoid device sync